guide/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,366 @@
+
+To prevent the recovery path from kicking in and refreshing the timelock, a UTXO has to be spent and re-deposited back into the wallet. Please refer to the [custom spending conditions page](https://bitcoin.design/guide/how-it-works/custom-spending-conditions/) for more information about how timelocks work. 
+ 
+
+##### Preventing recovery path activation
+
+So if Bob has three UTXOs in his wallet which he received at different times, he actually needs to refresh three different timelocks. This can quickly become cumbersome and costly, as every refresh transaction implies on-chain transaction fees.
+
+To reduce this complexity, our application offers a "unified" timelock reset experience by default. This simply means that it uses the UTXO that is closest to being unlocked as the trigger to start the process. In the background, however, all UTXOs are automatically batched together. 
+
+This allows us to show one timelock, which makes it easier to understand for users and is more in line with their expectations. An added benefit of this approach is that, as a result, Bob has only one UTXO in his wallet after the timelock refresh.
+
+{% include image-gallery.html pages = page.images_prevent-unlock %}
+
+To give users additional flexibility, our application allows users to selectively manage the refresh behavior for individual UTXOs. This is similar to the [coin control](https://bitcoin.design/guide/glossary/#coin-control) functionality available in many wallet applications. 
+
+##### Refreshing an activated recovery path
+
+Just because a recovery path has been activated does not mean that it's too late. Bob can always refresh it simply by creating the same self-transfer outlined above.
+
+{% include image-gallery.html pages = page.images_path-reset %}
+
+
+### Wallet recovery
+
+
+##### Switching to a new phone or computer
+
+One of the most likely reasons to recover a wallet is that users switch to a new phone or computer. This means that they have to re-install the wallet application and recover the wallet. As long as Alice is using the same signing device as before, all she has to do is to import the wallet in the same way as described in the co-signer onboarding section above. 
+
+##### Replacing a signing device (using the same private key)
+
+If Bob wants to replace one of the signing devices he can use the new device with the same wallet, as long as it uses the same private key as the original one. This might become necessary because the original device is broken or Bob just wants to use a newer model. To do that, he will need to:
+
+1. Restore the private key on the new signing device, using the seed phrase backup.
+2. Pair the new signing device, since it's extended public key has changed.
```

Comment:
That's not how it works, is it? The same private key should result in the same XPUB. If you change the XPUB, you'll get a different wallet with different addresses and will not be able to access the funds, no? Pretty sure you need to basically set up a completely new wallet and transfer funds over. FROST should be able to do what you're describing here, AFAIK.

Reply:
You're right about the XPUB 😬.

Reply:
Updated the text and the mockups